src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,96 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+
+import { getNumbers, getItemsToShowIndex } from '../../utils';
```

Comment:
I cannot seem to find implementation of the getItemsToShowIndex either in this pr nor on mates original branch

Reply:
Ahh forgot to commit utils file.